romdisk/strutils.asm

```diff
@@ -360,6 +362,66 @@ _parse_upper_hex_digit:
         sub 'A' - 10 ; CY will be reset
         ret
 
+        ; Convert a 16-bit value to ASCII (u16 decimal)
+        ; Parameters:
+        ;       HL - 16 bit value to display
+        ;       DE - String destination. It must have at least 5 free bytes to write the ASCII result.
+        ;
+        ; Returns:
+        ;       DE - location of last character +1
+        ; Alters:
+        ;       A, DE
+        PUBLIC word_to_ascii_dec_u16
+word_to_ascii_dec_u16:
+        push bc
```

Comment:
There might be a faster way we could look into, like for example converting to BCD (double dabble algorithm?) or using fast divide (16 shifts). We can keep it like this for the moment, but maybe you can add a comment so that we don't forget :smile:

Reply:
It's larger code, but turns out to be much better at being extended to 32 bits. Which I've done. It's now 32 bit...